### PR TITLE
Added nmap version and Dockerfile

### PR DESCRIPTION
**Pull request by wevans77**
Created nmap.py that uses nmap instead of ping. Couldn't find version of nmap module for Python that properly implemented "-sn" for network scan so calling nmap directly.

### CODE REVIEW

File: POC/scanner.py [hunk 1]
```diff
@@ -1,58 +1,123 @@
...
+            hostname = hostnameElem.attrib['name']
 
-for sent, received in result:
+        client_ip = host.find('./address/[@addrtype=\'ipv4\']').attrib['addr']
+
+        macElem = host.find('./address/[@addrtype=\'mac\']')
+
+        if macElem!=None:
+            client_mac = macElem.attrib['addr']
+            client_vendor = macElem.attrib['vendor']
```
Comment (masta-bhawk) [inline POC/scanner.py]:
```suggestion
            try:
                client_vendor = macElem.attrib['vendor']
            except:
                client_vendor = 'Unknown'
```

Comment (masta-bhawk) [reply POC/scanner.py]: If the mac vendor doesn't exist in the db, we need to not fail.  (Yep I have one.)

File: README.md [hunk 2]
```diff
@@ -1 +1,7 @@
-# NetBitch
\ No newline at end of file
+# NetBitch
+
+To compile and run docker image:
+
+`docker build -t netbitch .`
+
+`docker run --net=host netbitch`
```
Comment (masta-bhawk) [inline README.md]: We gotta figure out how to do this without --net=host

Comment (wevans77) [reply README.md]: Yes, agreed. Although until we know how to do this, Docker won't work. At least with a Dockerfile here we can experiment.

**Comment by wevans77** [comment]
Good catch on the try around mac address. Does the rest of the nmap code work for you? Thinking we might have to support multiple implementations as long as we keep the structure the same should be okay for now

File: README.md (hunk 2 above)
Comment (masta-bhawk) [reply README.md]: agreed

**Pull request merged**